system/lib/compiler-rt/lib/sanitizer_common/sanitizer_posix_libcdep.cpp

```diff
@@ -277,6 +277,11 @@ bool SignalContext::IsStackOverflow() const {
 #endif  // SANITIZER_GO
 
 bool IsAccessibleMemoryRange(uptr beg, uptr size) {
+#ifdef __wasm__
+  // Avoid pulling in __sys_pipe for the trick below, which doesn't work on
```

Comment:
I don't see any `__wasm__` checks before this - but lots of ifdefs on `SANITIZER_EMSCRIPTEN`. Maybe that's what we should use?